trello/features/smoke/postsmokeboard.feature

```diff
@@ -0,0 +1,13 @@
+# Created by Oscar Lopez at 1/27/2020
+Feature: Board's Test
+  # As a regular user, Manages the board, so that creates a nuew board.
+  Scenario: Creates new board with a name
+    Given Sends a POST request to https://api.trello.com/1/boards/
+      | key  | value    |
+      | name | newBoard |
+
+    And Sends request
+    And Should return "200" status code as response
+    Then Should return a body response "jsonfiles/response_body_board.json"
```

Comment:
please fix this line to something like this:
  Then Should return status code "200"
  And validate response body
  And validate schema